packages/web-components/src/components/ic-classification-banner/ic-classification-banner.tsx

```diff
@@ -37,7 +37,13 @@ export class ClassificationBanner {
   @Prop() upTo?: boolean = false;
 
   render() {
-    const { classification, inline, country, upTo, additionalSelectors } = this;
+    const { inline, upTo } = this;
+
+    // In case of unrecognized props, fallback to default 
+    let { country, additionalSelectors, classification } = this;
```

Comment:
@GCHQ-Developer-530 I ran both storybook and a local reproduction repo (https://github.com/evenstensberg/banner-repro-ic). In my own repo, the behaviour is as expected.

In storybook I got the same bug as you, and I've added some checks that will default to the default props, if the props are wrong or empty.

There's one case left, and that is to check if a string prop is boolean. If the prop such as `classification` is passed as boolean, the classification is `true`. Let me know if you want me to add check if boolean is passed.

Reply:
I think this ticket was just to deal with the prop being undefined, so it's safe to leave out the boolean work for now